auth.go

```diff
@@ -243,12 +278,15 @@ type Central struct {
 	NewSessionExpiresAfter time.Duration
 	DisablePasswordReuse   bool
 	PasswordExpiresAfter   time.Duration
+	MaxFailedLoginAttempts int // only applies if EnableAccountLocking is true
+	EnableAccountLocking   bool
 	userStoreMergeTicker   *time.Ticker
 	mergeCount             int
 	ldapUsed               bool
 	ldapMergeTickerSeconds time.Duration
 	tickerStopReq          chan bool
 	tickerStopResp         chan bool
+	LockingPolicy          LockingPolicy
```

Comment:
Follow the convention of the local code, and place public members before private